CMakeLists.txt

```diff
@@ -481,6 +481,17 @@ if(HELICS_BUILD_CONFIGURATION STREQUAL "PI")
     target_link_libraries(helics_base INTERFACE atomic)
 endif()
 
+# Add to list of enabled vcpkg features based on core type
+if(HELICS_ENABLE_ZMQ_CORE)
+    list(APPEND VCPKG_MANIFEST_FEATURES "zeromq")
```

Comment:
where does this list get used

Reply:
In theory when building using vcpkg it can specify which features are enabled, so it will automatically built the right subset based on which have been enabled/disabled. I think to actually make that work the related CMake options and appending to VCPKG_MANIFEST_FEATURES needs to be reordered so they appear at the top of CMakeLists.txt, prior to setting the CMAKE_TOOLCHAIN_FILE to vcpkg.cmake and the project() command.